origins_workflow/tests/src/Nightwatch/Tests/moderationSidebar/02.draftToNeedsReview.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  '@tags': ['origins', 'origins_workflow', 'debug'],
+
+  before: function (browser) {
+    // Resize default window size.
+    browser.resizeWindow(1600, 2048);
+
+    // Login as an editor.
+    browser
+      .drupalLogin({ name: process.env.NW_TEST_USER_PREFIX + '_editor', password: process.env.TEST_PASS })
+      .drupalRelativeURL('/node/add/article');
+
+    // Create a minimal article, populating mandatory fields only.
+    // SHS widget isn't very friendly for form values so we need to simulate
+    // actual click events on certain DOM elements.
+    // '12' == Motoring theme.
+    browser.click('select[id="edit-field-subtheme-shs-0-0"]');
+    browser.click('select[id="edit-field-subtheme-shs-0-0"] option[value="12"]');
```

Comment:
I found you can use either a selector like `#edit-field-subtheme-shs-0-0` or the one here. Some anecdotal posts online suggested the attribute based selector caused fewer issues at runtime, although I haven't see much difference myself. These are just here because they were the last thing I tried when debugging something slightly related to it, left in because they work and seemed silly to pull them out for tiny bit of brevity